layers/+lang/python/packages.el

```diff
@@ -28,6 +28,8 @@
     company
     cython-mode
     dap-mode
+    ;; We are using a fork until pet is prefering ipython as default shell (https://github.com/wyuenho/emacs-pet/pull/56)
+    (pet :location (recipe :fetcher github :repo "smile13241324/emacs-pet"))
```

Comment:
Does Spacemacs always prefer ipython? Before this PR, did it also change all the package variables that need a path to a python executable, such as `lsp-pyright-python-executable-cmd`?